.travis.yml

```diff
@@ -1,84 +0,0 @@
-branches:
-  only:
-    - master
-    - 2.x
-    - /^\d+\.\d+\.\d+/  # npm version tags
-
-language: node_js
-node_js:
-  - "10"
-
-addons:
-  chrome: stable
-
-cache:
-  yarn: true
-
-env:
-  secure: IVCX1o8nMAKAKxj2cXE6a2A8xmnt3Gj72ocyjT1vhZutEJ1z7wu3Xl/r6Q0fhlaux3+ZNz/KV6Lax6NW8KgC/Wo1+B5SF2RWMLGX+exglf5zUXT3wzaOq8LcEyQ7SGfsuovHa8ij3dMM4UWjl5lE3cM1M1KjxlekniUF9nFxZd4=
-
-jobs:
-  fail_fast: true
-  allow_failures:
-    - env: SCRIPT="test:one ember-beta"
-    - env: SCRIPT="test:one ember-canary"
-    - env: SCRIPT=test:fastboot
-
-  include:
-    # runs linting and tests with current locked deps
-    - stage: 'Basic Tests'
-      name: 'Lint'
-      env: SCRIPT=lint
-
-    - name: 'Fixed Dependencies'
-      env: SCRIPT="test"
-
-    - name: 'Floating Dependencies'
-      env: SCRIPT="test"
-      install: yarn install --no-lockfile
```

Comment:
we don't seem to test this case anymore?

Reply:
added back in